app/code/Magento/Reports/Model/ResourceModel/Report/AbstractReport.php

```diff
@@ -451,9 +451,7 @@ protected function _getTZOffsetTransitions($timezone, $from = null, $to = null)
                     : (new \DateTime($from))->getTimestamp();
             }
 
-            $to = $to instanceof \DateTimeInterface
-                ? $to
-                : new \DateTime($to);
+            $to = $to instanceof \DateTimeInterface ? $to : new \DateTime($to ?? 'now');
```

Comment:
Hello @anzin 
This has already fixed in [this PR](https://github.com/magento/magento2/pull/34671)
Please roll back this file